README.md

```diff
@@ -38,11 +38,9 @@ linux	/boot/vmlinuz-6.5.0-10-generic nomodeset root=UUID=c5605a23-05ae-4d9d-b65f
 ```
 * Step #3 below will automatically add `nomodeset` to the GRUB config so that no additional intervention is needed
 
-2. Clone this repo to get scripts that will be used throughout this README.
-
-```bash
-git clone https://github.com/canonical/tdx.git
-```
+2. Download source files from this repository that will be used throughout this README. This can be done by downloading an asset
```

Comment:
I would shorten the text like this : Download this repository ... or by cloning it at the appropriate tag.